server/src/main/java/com/objectcomputing/checkins/services/skills/SkillController.java

```diff
@@ -0,0 +1,110 @@
+package com.objectcomputing.checkins.services.skills;
+
+import io.micronaut.http.HttpResponse;
+import io.micronaut.http.HttpStatus;
+import io.micronaut.http.MediaType;
+import io.micronaut.http.annotation.*;
+import io.micronaut.security.annotation.Secured;
+import io.micronaut.security.rules.SecurityRule;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.Nullable;
+import javax.inject.Inject;
+import javax.validation.Valid;
+import java.net.URI;
+import java.util.List;
+import java.util.UUID;
+
+@Controller("/skill")
+@Secured(SecurityRule.IS_ANONYMOUS)
+@Produces(MediaType.APPLICATION_JSON)
+@Tag(name="skill")
+public class SkillController {
+
+    private SkillRepository skillRepository;
+
+    @Inject
+    private SkillServices skillsService;
+
+    public void setSkillRepo(SkillRepository skillsRepository) {
+        this.skillRepository = skillsRepository;
+    }
+
+    /**
+     * Create and save a new skill.
+     *
+     * @param skill
+     * @return
+     */
+
+    @Post(value = "/")
+    public HttpResponse<Skill> createASkill(@Body @Valid Skill skill) {
+        Skill newSkill = skillsService.saveSkill(skill);
+
+        if (newSkill == null) {
+            return HttpResponse.status(HttpStatus.valueOf(409), "already exists");
+        } else {
+            return HttpResponse
+                    .created(newSkill)
+                    .headers(headers -> headers.location(location(newSkill.getSkillid())));
+        }
+    }
+
+    /**
+     * Load the current skills into checkinsdb.
+     *
+     * @param skillslist
+     * @return
+     */
+
+    @Post("/loadskills")
```

Comment:
Normally, I would say that we shouldn't have a verb in a URL. REST services as a rule are based around the entity and the "action" is driven by the HTTP command being used.  So, I would have probably made a single @Post service like the one below and not separated them out into two. However, since this is not a service that is intended for use by the frontend, but rather for convenience for now...I think it's ok.  Eventually we probably want to include the "load" of skills into a Flyway migration instead, at which point we can remove the loadSkills service.